topics/thelounge/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: thelounge-theme
+display_name: The Lounge
+github_url: https://github.com/thelounge/lounge
+logo: thelounge.png
+related: irc, irc-client, chat, javascript
+released: February 12, 2016
+short_description: The Lounge is a modern web IRC client that designed for self-hosting.
+topic: thelounge
+url: https://thelounge.chat
+---
+The Lounge is a modern web IRC client designed for self-hosting. It runs in the browser, works across platform, and keeps IRC connections open when clients are closed. It is built with JavaScript and Node.js.
```

Comment:
`works across platform` - maybe `works across multiple platforms` or `works cross-platform`?

After "IRC", you could put "(Internet Relay Chat)" in case someone ends up on the topic page without first being familiar with IRC.

Reply:
Will do!